Sources/Extensions/ColletionExtension.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ColletionExtension.swift
+//  HandySwift iOS
+//
+//  Created by Stepanov Pavel on 08/07/2018.
+//  Copyright © 2018 Flinesoft. All rights reserved.
+//
+
+import Foundation
+
+extension Collection {
+    
+    /// Returns an element with the specified index or nil if the element does not exist
+    ///
+    /// - Parameters:
+    ///   - safe: The index of the element
+    public subscript (safe index: Index) -> Element? {
```

Comment:
- [ ] Please also check this line with SwiftLint installed (the whitespace between `subscript` and `(safe` should show a warning).
- [ ] We already wanted to add such a subscript to this project with some colleges and discussed, how this subscript should be named. We agreed on the subscript name `(try` instead of `(safe`. If you're okay with that name, too, could you rename your implementation?